packages/matchbox/src/components/Popover/Popover.tsx

```diff
@@ -80,9 +132,9 @@ const Popover = React.forwardRef(function Popover(props, ref) {
   function handleOutsideClick(e) {
```

Comment:
`Parameter 'e' implicitly has an 'any' type`

Was trying to add types but not sure what it should be, probably fine without it though